Dockerfile

```diff
@@ -0,0 +1,35 @@
+# This dockerfile is used to test containerd within a container
+#
+# usage:
+# 1.) docker build -t containerd-test -f Dockerfile .
```

Comment:
s/Dockerfile/Dockerfile.test/ , as this Dockerfile is not suitable for other purposes

Reply:
In a follow up we could add another stage which runs `script/setup/install-dev-tools`. That would make this Dockerfile usable for just about every project task. Then do you think we could keep it `Dockerfile` or `Dockerfile.dev` ?